charts/incubator/misskey/templates/ConfigMap.yaml

```
@@ -0,0 +1,172 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: misskeyconfig
+data:
+  default.yml: |-
+    #━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+    # Misskey configuration
+    #━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+    #   ┌─────┐
+    #───┘ URL └─────────────────────────────────────────────────────
+
+    # Final accessible URL seen by a user.
+    # url: https://example.tld/
+
+    # ONCE YOU HAVE STARTED THE INSTANCE, DO NOT CHANGE THE
+    # URL SETTINGS AFTER THAT!
+
+    #   ┌───────────────────────┐
+    #───┘ Port and TLS settings └───────────────────────────────────
+
+    #
+    # Misskey supports two deployment options for public.
+    #
+
+    # Option 1: With Reverse Proxy
+    #
+    #                 +----- https://example.tld/ ------------+
+    #   +------+      |+-------------+      +----------------+|
+    #   | User | ---> || Proxy (443) | ---> | Misskey (3000) ||
+    #   +------+      |+-------------+      +----------------+|
+    #                 +---------------------------------------+
+    #
+    #   You need to setup reverse proxy. (eg. nginx)
+    #   You do not define 'https' section.
+
+    # Option 2: Standalone
+    #
+    #                 +- https://example.tld/ -+
+    #   +------+      |   +---------------+    |
+    #   | User | ---> |   | Misskey (443) |    |
+    #   +------+      |   +---------------+    |
+    #                 +------------------------+
+    #
+    #   You need to run Misskey as root.
+    #   You need to set Certificate in 'https' section.
+
+    # To use option 1, uncomment below line.
+    port: {{ .Values.service.main.ports.main.targetPort }} # A port that your Misskey server should listen.
```

```suggestion
    port: {{ .Values.service.main.ports.main.port }} # A port that your Misskey server should listen.
```

This will make it both easier for Scale users to change port (target port is double hidden, port not) 
and you can remove targetPort from values.yaml (it will default to `port`

---

Thanks for the suggestion.